doc/user/content/get-started.md

```diff
@@ -61,192 +64,304 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
+    Unlike a view, the results of a materialized view are persisted and incrementally updated **in durable storage** as new data arrives. Because the storage layer is shared across all clusters, materialized views allow you to **share** data between different environments.
 
-### Temporal filters
+1. To see the results:
 
-In Materialize, [temporal filters](/sql/patterns/temporal-filters/) allow you to define time-windows over otherwise unbounded streams of data. This is useful to model business processes or simply to limit resource usage.
+    ```sql
+    SELECT * FROM item_summary ORDER BY revenue DESC LIMIT 5;
+    ```
 
-1. In this case, we need to filter out rows for auctions that are still ongoing:
+    <br>
+
+    <details>
+      <summary>
+        🤘 <b>Can we optimize the join?</b>
+      </summary>
+      <code>
+        materialize=> CREATE INDEX item_idx ON items (id);
+
+        materialize=> EXPLAIN SELECT * FROM item_summary ORDER BY revenue DESC LIMIT 5;
+                                   Optimized Plan
+        ---------------------------------------------------------------------
+         Explained Query:                                                   +
+           Finish order_by=[#4 desc nulls_first] limit=5 output=[#0..=#4]   +
+             Reduce group_by=[#3, #4] aggregates=[sum(#2), sum(#1), sum(#0)]+
+               Project (#1..=#3, #5, #6)                                    +
+                 Join on=(#0 = #4) type=differential                        +
+                   ArrangeBy keys=[[#0]]                                    +
+                     Get materialize.public.item_purchases                  +
+                   ArrangeBy keys=[[#0]]                                    +
+                     Get materialize.public.items                           +
+                                                                            +
+         Used Indexes:                                                      +
+           - materialize.public.item_purchases_idx                          +
+           - materialize.public.item_idx                                    +
+      </code>
+    </details>
+
+[//]: # "NOTE(morsapaes) Suggestions for the section name? I strongly dislike this one"
+### Work with time
+
+In streaming, time keeps ticking along — unlike in batch, where data is "frozen" at query time. [Temporal filters](/sql/patterns/temporal-filters/) allow you to define time-windows over otherwise unbounded streams of data.
+
+1. Create a view `item_summary_5min` that  to items that had orders in the past 5 minutes (or, 300000 miliseconds).
 
     ```sql
-    CREATE MATERIALIZED VIEW winning_bids AS
-        SELECT * FROM highest_bid_per_auction
-        WHERE end_time < mz_now();
+    CREATE VIEW item_summary_5min AS
```

Comment:
May be overkill stylistically, but should we use a consistent prefix (like `qck_` used in source creation) on all quickstart objects? 

For the docs only approach we can create funnels in our analytics based on the exact names in the docs, but thinking about a embedded version where we may show incremented (or other unique) variations and we'll want to be able to sniff these out without false positives. It would be possible for users to organically have something called `item_summary` in the future, for instance.

Reply:
There's room for discussion in terms of how we handle namespacing for the quickstarts, and we should definitely take into account what works best for analytics. The fact that not all objects are tied to a cluster makes it harder to create a self-contained environment that 1) isn't overloaded with prefixes and fully qualified names and 2) can be spun down cleanly. 😢 



It's easier to iterate on these details in [Notion](https://www.notion.so/materialize/Get-started-5d4fcf7893474b529b3626169e6d792e), so let's regroup there!